msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CompileEntitlementsTaskBase.cs

```diff
@@ -69,12 +69,12 @@ PString MergeEntitlementString (PString pstr, MobileProvision profile, bool expa
 
 			if (profile == null) {
 				if (!warnedTeamIdentifierPrefix && pstr.Value.Contains ("$(TeamIdentifierPrefix)")) {
-					Log.LogWarning (null, null, null, Entitlements, 0, 0, 0, 0, "Cannot expand $(TeamIdentifierPrefix) in Entitlements.plist without a provisioning profile.");
+					Log.MTError (7068, "Cannot expand $(TeamIdentifierPrefix) in Entitlements.plist without a provisioning profile.", Entitlements);
```

Comment:
As Sebastien said, not sure we want to make that old warning an error. You need to introduce some MTWarning (: